results/ttest_hyphalnetwork_distances.R

```diff
@@ -0,0 +1,38 @@
+#install.packages('dplyr')
+#install.packages('MASS')
+if(!require(dplyr))
+  install.packages('dplyr')
+if(!require(MASS))
+  install.packages("MASS")
+
+library(MXM)
+
+
+syn=MXM::synapseLogin()
+data = MXM::querySynapseTable("syn22279826")
+
+
+#independent t test
+community = MPNST %>% filter(net2_type == 'community')
```

Comment:
Doing a single test will not work - t-tests rely on distributions. Group all tumors together and compare them to the values for each xenograft, for each community. Each community should have a single statistic.